mPowerSDK/Dashboard/APHWebViewStepViewController.m

```diff
@@ -48,7 +54,40 @@ @interface APHWebviewViewController () <UIWebViewDelegate>
 
 @end
 
-@implementation APHWebviewViewController
+@implementation APHWebViewStepViewController
+
++ (instancetype)new {
```

Comment:
Not keen on the idea of overriding new like this. General expectation is that [X new] is equivalent to [[X alloc] init]. Can we use a different name?